airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -160,7 +160,7 @@ public CheckConnectionRead checkSourceConnectionFromSourceIdForUpdate(SourceUpda
     final SourceConnection updatedSource = configurationUpdate.source(sourceUpdate.getSourceId(), sourceUpdate.getConnectionConfiguration());
 
     final ConnectorSpecification spec = getSpecFromSourceDefinitionId(updatedSource.getSourceDefinitionId());
-    jsonSchemaValidator.validate(spec.getConnectionSpecification(), updatedSource.getConfiguration());
+    jsonSchemaValidator.ensure(spec.getConnectionSpecification(), updatedSource.getConfiguration());
```

Comment:
i think this was just wrong. we want it to throw here if it doesn't pass validation.